build/ublue-os-luks/luks-enable-tpm2-autounlock

```diff
@@ -1,57 +1,65 @@
 #!/bin/bash
 ## setup auto-unlock LUKS2 encrypted root on Fedora/Silverblue/maybe others
-set -u
+set -eou pipefail
 
 [ "$UID" -eq 0 ] || { echo "This script must be run as root."; exit 1;}
 
 read -p "This will modify your system and enable TPM2 auto-unlock of your LUKS partition! Are you sure you've read the script and are good with this? " -n 1 -r
```

Comment:
Should we remove "you sure you've read the script and" and instead state that we only use PCR7 which and give some idea what will cause the auto-unlock to fail and require a manual unlock?

Reply:
We should indicate that this will bind to secureboot state via PCR7. That will give a user some idea on what this is doing.

And probably specify that this is using systemd-cryptenroll so a user can look up information about it.

Also, clearly users are not inspecting the script prior to running so I'm unsure if that warning is actually serving any purpose right now.